typescript/packages/subsurface-viewer/src/layers/axes2d/axes2DLayer.ts

```diff
@@ -62,15 +62,13 @@ const zDepthAxesAxis = zDepthAxesBackground + 999;
 const zDepthAxesTicksAndLabels = zDepthAxesAxis;
 
 const tickLineLength = 10;
-const pixelScale = 8;
 
 export interface Axes2DLayerProps extends ExtendedLayerProps {
     marginH: number;
     marginV: number;
     formatLabelFunc?: (x: number) => string;
     labelColor?: Color;
-    labelFontSize?: number;
```

Comment:
This is a breaking change. Suggest keeping the old prop and document it as deprecated.